pose_evaluation/metrics/.gitignore

```diff
@@ -0,0 +1 @@
+tests/
```

Comment:
- file does not end in a new line
- why not use `tempfile` instead of a directory?

Reply:
Because I wanted the option to visually inspect the results for correctness, e.g. 
![image](https://github.com/user-attachments/assets/0a774cd3-a9db-46b2-befc-72b312ba4dcb)

Could I do that with a tempfile somehow? I'll look into it

Reply:
no need then. maybe call the directory `temp` or `test_results` or something - `tests` just imply that there are tests there

Reply:
Good point. Fixed.